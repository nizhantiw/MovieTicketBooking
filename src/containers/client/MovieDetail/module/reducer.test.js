import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme, { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer'
import reducer from './reducer'
configure({adapter: new Adapter()});

it("renders correctly", () => {
  const wrapper = shallow(
    <reducer />
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders a snapshot', () => {
  const tree = renderer.create(<reducer/>).toJSON();
  expect(tree).toMatchSnapshot();
});



