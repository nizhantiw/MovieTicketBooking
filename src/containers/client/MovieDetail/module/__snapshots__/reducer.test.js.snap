// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `<reducer />`;

exports[`renders correctly 1`] = `ShallowWrapper {}`;
